### PR TITLE
Calendar view

### PR DESCRIPTION
**Pull request by joenas**
TODO:

- [x] Translate `CALENDAR_NO_CONTENT` and `CALENDAR_FULL_DAY`
- [x] Should deps be in `guide-app/package.json` or not?
- [x] Add new strings to Lund
- [x] Analytics

Regarding dependencies we had a discussion and agreed on **not** having them in `guide-app/package.json`, mostly because it's confusing. 
An argument would be to only have "linkable" packages in the actual cities and the "pure JS" ones in `guide-app` but that could easily be more confusing and involves just as much work, if not more, so skipping that.

### CODE REVIEW

File: guide-app/src/components/shared/CalendarDatePicker.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      fontStyle: "normal",
+      color: Colors.black26
+    }
+  ])
+});
+
+type Props = {
+  chosenDate: Date,
+  currentLanguage: string,
+  // TODO: :)
```
Comment (worldeggplant) [inline guide-app/src/components/shared/CalendarDatePicker.js]: TODO
Please document what to do if leaving it in.

File: guide-app/src/components/shared/CalendarEvent.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+  },
+  listItemHoursView: {
+    borderRadius: 4,
+    backgroundColor: Colors.white,
+    paddingTop: 5,
+    paddingBottom: 4,
+    paddingHorizontal: 15,
+    shadowRadius: 12,
+    shadowColor: "black",
+    shadowOpacity: 0.16
```
Comment (worldeggplant) [inline guide-app/src/components/shared/CalendarEvent.js]: You're not including Android's `elevation` prop here.

File: guide-app/src/components/shared/CalendarEvent.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    location,
+    name,
+    slug,
+    dateStart,
+    dateEnd
+  } = event;
+  const image = imageUrl ? { uri: imageUrl } : defaultImage;
+
+  const entities = new XmlEntities();
+  const decodedLocationTitle = entities.decode(location.title);
```
Comment (worldeggplant) [inline guide-app/src/components/shared/CalendarEvent.js:122]: Do we want to do the decoding in this component, or move this out & elsewhere (like in a reducer?)? Seems like something more tied to the API than this particular component.

File: guide-app/src/reducers/eventReducer.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      // action.events.forEach(g => {
+      //   const index = items.findIndex(item => item.id === g.id);
+      //   if (index >= 0) {
+      //     // replace
+      //     items[index] = g;
+      //   } else {
+      //     // push
+      //     items.push(g);
+      //   }
+      // });
```
Comment (worldeggplant) [inline guide-app/src/reducers/eventReducer.js]: Comment

**Review by worldeggplant — APPROVED** [review]

File: guide-app/src/components/shared/CalendarDatePicker.js (hunk 1 above)
Comment (joenas) [reply guide-app/src/components/shared/CalendarDatePicker.js]: Fixeed

File: guide-app/src/reducers/eventReducer.js (hunk 4 above)
Comment (joenas) [reply guide-app/src/reducers/eventReducer.js]: Yeah this was one of the things I wanted to have a discussion about, but for now I think I'll just remove it and we can revisit

File: guide-app/src/components/shared/CalendarEvent.js (hunk 3 above)
Comment (joenas) [reply guide-app/src/components/shared/CalendarEvent.js:122]: Hm yeah. Is the reducer the best place you think?

File: guide-app/src/reducers/eventReducer.js (hunk 4 above)
Comment (joenas) [reply guide-app/src/reducers/eventReducer.js]: Added some comments :)

**Pull request merged**